target/linux/ath79/dts/qca9557_buffalo_bhr-4grv2.dts

```diff
@@ -103,15 +103,101 @@
 &mdio0 {
 	status = "okay";
 
-	phy0: ethernet-phy@0 {
+	switch0@10 {
```

Comment:
Why @10? Where is the "10" coming from?

Reply:
@Ansuel probably has the answer. Most of this is copy/paste from his ipq work. I will also note @blocktrron 's ath79 DSA work (for devices using the builtin switch) uses the same number.

Reply:
switch reg it comes from qca8k documentation example and is needed as if i remember correctly it did cause some problem with dsa (having set it to 0)